poolmgr/gp.go

```diff
@@ -311,12 +344,35 @@ func (gp *GenericPool) specializePod(pod *v1.Pod, metadata *api.ObjectMeta) erro
 
 	// get function run container to specialize
 	log.Printf("[%v] specializing pod", metadata.Name)
-	specializeUrl := gp.getSpecializeUrl(podIP)
 
 	// retry the specialize call a few times in case the env server hasn't come up yet
 	maxRetries := 20
+
+	fn, err = gp.fissionClient.Functions(metadata.Namespace).Get(metadata.Name)
+	if err != nil {
+		return err
+	}
+
+	loadReq := fission.FunctionLoadRequest{
+		FilePath:     filepath.Join(gp.sharedMountPath, "user"),
```

Comment:
can you keep the string `"user"` in variable somewhere instead of hard-coding it twice?